doc/sphinx/source/code.rst

```diff
@@ -0,0 +1,111 @@
+.. _Code:
+
+Code
+======================================
+
+This section contains the code documentation. The subsections represent
+the different API objects, typedefs, and enumerations.
+
+
+API Objects
```

Comment:
Again with 'words are hard'. 'API Objects' makes me think of 'objects' where this is perhaps more broadly the libCEED objects and interface? I'm not sure.

Reply:
Public Types (if including enums) or Public Classes?

Reply:
Currently this has the all functions - User, Backend Developer, and Frontend Developer (really I should have called the last one something like 'Library Developer') even though not all of these functions are exposed in `ceed.h`. (Doxygen has the same thing)

Reply:
Renamed